src/layouts/app/administrative/users/edit/index.tsx

```diff
@@ -1,6 +1,180 @@
+          <Grid item mb={2}>
+            <Card>
+              <CardHeader title={<Typography variant="h4">Main</Typography>} />
+              <Divider />
+              <CardContent>
+                <Box>
+                  <FInput
+                    required
+                    type="text"
+                    size="small"
+                    name="firstName"
+                    label={<strong>First Name</strong>}
+                  />
+                </Box>
+                <Box pt={2}>
+                  <FInput
+                    type="text"
+                    size="small"
+                    name="middleName"
+                    label={<strong>Middle Name</strong>}
+                  />
+                </Box>
+                <Box pt={2}>
+                  <FInput
+                    required
+                    type="text"
+                    size="small"
+                    name="lastName"
+                    label={<strong>Last Name</strong>}
+                  />
+                </Box>
+                <Box pt={2}>
+                  <FSelect
+                    fullWidth
+                    required
+                    size="small"
+                    name="suffix"
+                    label={<strong>Suffix</strong>}
+                    getFieldValue={(({ value }: AnyObject) => value)}
+                    prepareValue={(value: string | number) => ({ value, label: value })}
+                    options={['Jr.', 'Sr.', '2nd', '3rd', 'II', 'III', 'IV', 'V'].map(option => ({ value: option, label: option }))}
+                  />
+                </Box>
+              </CardContent>
+            </Card>
+          </Grid>
+          <Grid item>
+            <Card>
+              <CardHeader title={<Typography variant="h4">Role</Typography>} />
+              <Divider />
+              <CardContent>
+                <Box>
+                  <FAsyncSelect
+                    multiple
+                    required
+                    fullWidth
+                    size="small"
+                    name="roles"
+                    loadingText="LOADING"
+                    filterSelectedOptions
+                    loadOptions={getRolesMemo}
+                    label={<strong>Roles</strong>}
+                    getOptionLabel={({ name }) => name}
+                    prepareValue={(value: AnyObject) => value}
+                    getFieldValue={(value) => value}
+                    isOptionEqualToValue={(option, value) => option.name === value.name}
+                  />
+                </Box>
+                <Box mt={2}>
+                  <FInput
+                    required
+                    type="text"
+                    size="small"
+                    name="email"
+                    label={<strong>Last Name</strong>}
+                  />
+                </Box>
+              </CardContent>
+            </Card>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Box>
```

Comment:
Button should have 16px offset to bottom and content should contain all fee height